patches/0109-lavf-dnn-enable-libtorch-backend-support-FRVSR-model.patch

```diff
@@ -0,0 +1,132 @@
+From 6edc0e9fb081886771c27086dcc08d0af9682748 Mon Sep 17 00:00:00 2001
+From: Ting Fu <ting.fu@intel.com>
+Date: Wed, 29 Jun 2022 07:46:44 +0000
+Subject: [PATCH 4/4] lavf/dnn: enable libtorch backend support FRVSR model
+
+Signed-off-by: Ting Fu <ting.fu@intel.com>
+---
+ libavfilter/dnn/dnn_backend_torch.cpp | 52 +++++++++++++++++++++++----
+ 1 file changed, 45 insertions(+), 7 deletions(-)
+
+diff --git a/libavfilter/dnn/dnn_backend_torch.cpp b/libavfilter/dnn/dnn_backend_torch.cpp
+index fde0a022ae..cefbca068d 100644
+--- a/libavfilter/dnn/dnn_backend_torch.cpp
++++ b/libavfilter/dnn/dnn_backend_torch.cpp
+@@ -46,6 +46,8 @@ typedef struct THContext {
+     THOptions options;
+ } THContext;
+ 
++typedef enum {UNKNOWN_MODEL = -1, BASICVSR, FRVSR} ModelType;
```

Comment:
I still saved the BASICVSR enum, for potential use.
And I changed the UNSUPPORTED_MODEL into UNKNOWN_MODEL. Since the UNKOWN_MODEL may also be runnable.